xayn-ai/src/coi/system.rs

```diff
@@ -294,7 +288,7 @@ impl CoiSystem {
             .collect::<BTreeSet<_>>();
 
         let len = coi.key_phrases().len() + candidates.len();
-        let max_key_phrases = self.config.max_key_phrases.min(len);
+        let max_key_phrases = Configuration::MAX_KEY_PHRASES.min(len);
```

Comment:
The problem with this is that we cannot test with different values.

Reply:
i'll change it to a const generic to allow for testing with different values.